packages/vats/src/provisionPool.js

```diff
@@ -120,24 +115,47 @@ export const start = async (zcf, privateArgs) => {
   // @ts-expect-error vbank purse is close enough for our use.
   const fundPurse = E(poolBank).getPurse(poolBrand);
 
-  // Metrics
-  let totalMintedProvided = AmountMath.makeEmpty(poolBrand);
-  let totalMintedConverted = AmountMath.makeEmpty(poolBrand);
-  /** @type {import('@agoric/inter-protocol/src/contractSupport.js').MetricsPublishKit<MetricsNotification>} */
-  const { metricsPublisher, metricsSubscriber } = makeMetricsPublishKit(
-    privateArgs.storageNode,
-    privateArgs.marshaller,
-  );
-  const updateMetrics = walletsProvisioned => {
-    metricsPublisher.publish(
-      harden({
-        walletsProvisioned,
-        totalMintedProvided,
-        totalMintedConverted,
-      }),
+  const makeMetrics = () => {
```

Comment:
neat encapsulation. We may want to move it back into main scope for durability but we can evaluate that then.